appeals-service-api/src/schemas/appeals.js

```diff
@@ -11,10 +11,17 @@ const AppealSchema = new Schema(
         unique: true,
       },
     },
-    text: {
+    appellantName: {
       type: String,
       required: true,
     },
+    appellantEmail: {
+      type: String,
+      required: true,
+    },
+    planningApplicationNumber: {
+      type: Number,
```

Comment:
note: not required, this is something we need to discuss more I think

Reply:
Is this the UUID for the user?

Reply:
No, this appears to be something else - https://has-appeal-alpha.herokuapp.com/v7/application-number

Unless I have misunderstood. 

I'm using the `uuid` from Mongo as the user's `uuid` - maybe they will do all of this jazz from the planning application number though, which... would probably make more sense from a UX pov. Good Q

Reply:
Aaaaah, my mistake

Reply:
I don't have an answer to this right now essentially. It's been raised on teams and is under active discussion. For now, the uuid from mongo is the unique reference I'm storing. From a "front end" dev pov it's a simple thing to change. From a back end dev perspective, it's a can of worms that PINS need to open, discuss, and eat like an I'm a celebrity challenge. Sorry, I don't really know where I'm going with this metaphor.

Reply:
I think I get it... that's deep...

Reply:
> I don't have an answer to this right now essentially. It's been raised on teams and is under active discussion. For now, the uuid from mongo is the unique reference I'm storing. From a "front end" dev pov it's a simple thing to change. From a back end dev perspective, it's a can of worms that PINS need to open, discuss, and eat like an I'm a celebrity challenge. Sorry, I don't really know where I'm going with this metaphor.

I loved the metaphor. Can I be Biggins in this?

Reply:
Always :D haha